packaging/fedora/pack-source

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Do you need bash for real or just a habit?

Reply:
Needed for `trap .. EXIT` now.